content/boilerplates/trace-generation.md

```diff
@@ -0,0 +1,7 @@
+To see trace data, you must send requests to your service. The number of requests depends on Istio's sampling rate.
```

Comment:
This boilerplate breaks the ordered lists in your procedures. Do not use a boilerplate for any text that is duplicated. There are several things you need to consider when creating boilerplate content. Please just include the content as steps in each of the procedures.

Reply:
I have considered the context around the text. These *aren't* separate steps. This is further explanation of the first step which is to send traffic. It belongs in the same step.

Nothing about this breaks the ordered list in any fashion.  Note the 1 followed by the 2 below. 

![image](https://user-images.githubusercontent.com/21148125/53750250-1b74d400-3e5e-11e9-89c4-349b0536bd5f.png)